config/locales/forms/fr.yml

```diff
@@ -18,6 +18,7 @@ fr:
     messages:
       current_address: Vous devriez être en mesure de recevoir du courrier à cette
         adresse.
+      remember_device: NOT TRANSLATED YET
```

Comment:
Do we need this translated or will this fall back to the English?

Reply:
This should fallback to english.